forge-gui/res/cardsfolder/upcoming/paranormal_analyst.txt

```diff
@@ -0,0 +1,7 @@
+Name:Paranormal Analyst
+ManaCost:1 U
+Types:Creature Human Detective
+PT:1/3
+T:Mode$ ManifestDread | ValidPlayer$ You | TriggerZones$ Battlefield | Trigger$ TrigReturn | TriggerDescription$ Whenever you manifest dread, put a card you put into your graveyard this way into your hand.
+SVar:TrigReturn:DB$ ChangeZone | Defined$ TriggeredSource | Origin$ Graveyard | Destination$ Hand 
```

Comment:
TriggeredSource is not the correct defined, can't figure what it should be

Reply:
by now you should know to check corresponding `setTriggeringObjects` method ;)
`TriggeredNewCardLKICopy`